website/docs/r/eks_access_policy_association.html.markdown

```diff
@@ -13,6 +13,12 @@ Access Entry Policy Association for an EKS Cluster.
 ## Example Usage
 
 ```terraform
+resource "aws_eks_access_entry" "example" {
+  cluster_name  = aws_eks_cluster.example.name
+  principal_arn = aws_iam_role.example.arn
```

Comment:
The `aws_eks_access_entry.principal_arn` and `aws_eks_access_policy_association.principal_arn` need to be the same, but in your example here, one references an `aws_iam_role` while the other references a `aws_iam_user`, leading to a mismatch.